src/intents.js

```diff
@@ -18,7 +16,7 @@ function injectService (url, element, intent) {
   element.appendChild(iframe)
 
   // Keeps only http://domain:port/
-  const serviceOrigin = new URL(url).origin
+  const serviceOrigin = url.split('/', 3).join('/')
```

Comment:
`URL` does not exist in node, so tests were failing. After a couple of minutes looking for a polyfill, I gave up.